.travis.yml

```diff
@@ -9,8 +9,8 @@ cache:
 matrix:
     fast_finish: true
     include:
-        - php: 7.0
-        - php: 7.0
+        - php: '7.0'
```

Comment:
They have no control over it since it's just the YAML parser, but most PHP folks are probably safe since the version numbers aren't so tricky :stuck_out_tongue: